Formula/freetype.rb

```diff
@@ -34,6 +34,10 @@ def install
     system "./configure", "--prefix=#{prefix}", "--without-harfbuzz"
     system "make"
     system "make", "install"
+
+    chdir prefix do
```

Comment:
Would prefer to avoid stepping into the directory. We can do this from the buildpath easily enough:
```ruby
%W[#{bin}/freetype-config #{lib}/pkgconfig/freetype2.pc].each do |f|
  inreplace f, prefix, opt_prefix
end
```